std/regex/internal/tests.d

```diff
@@ -452,7 +452,7 @@ import std.uni : Escapables; // characters that need escaping
                     import std.stdio;
```

Comment:
That is optional now. Though it looks like it's the only `std.stdio` import in this module even though there are six more `debug(std_regex_test) writeln`, so maybe put `debug(std_regex_test) import std.stdio;`at the module level?

Reply:
I think CT testsuit is horribly slow to build so I wanted to print all errors originally. Make it an assert then to avoid getting bugs silently.